tests/foreman/ui/test_usergroup.py

```diff
@@ -80,18 +63,18 @@ def test_negative_create_usergroup_with_same_name(self, group_name):
         @Assert: Usergroup cannot be  created with existing name
 
         """
+        group_name = gen_string('alphanumeric')
         with Session(self.browser) as session:
-            make_usergroup(
-                session, org=self.organization.name, name=group_name)
+            make_usergroup(session, org=self.organization.name,
+                           name=group_name)
```

Comment:
Check answer in that PR
https://github.com/SatelliteQE/robottelo/pull/2928
Maybe that is same thing you are referring here